problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -181,5 +181,49 @@
 				real numbers.
 			\end{enumerate}
 		\end{solution}
+	
+		\prob Let $\vec u, \vec v \in \R^{n}$. In this problem, we will prove
+		$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm{\vec v}$, the \emph{Cauchy-Schwarz
+		inequality}.
+		\begin{enumerate}
+			\item Warm up: Suppose $\vec u, \vec v \in \R^{2}$ and $\vec u \cdot
+				\vec v = \Norm{\vec u}\Norm{\vec v}\cos \theta$ where $\theta$ is
+				the angle between $\vec u$ and $\vec v$. Show that
+				$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm{\vec v}$.
+
+			\item The result is immediate if one (or both) of $\vec u, \vec v$
```

Comment:
Make this: The Cachy-Schwartz inequality can also be proved using \emph{only} the algebraic definition of the dot product. Keep in mind the following facts which come from the algebraic definition: ... <list the linearity and norm properites you need>

Move the rest to subparts. Make the `u` or `v` = `0` case the first part.